packages/java/scripts/get-dev-server.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
Please check the headers.

Reply:
as you're introducing this file it should be Red Hat

Reply:
The code is mostly copied, so I think "Typefox and others" is correct.